deidentification/anonymizer.py

```diff
@@ -159,14 +161,19 @@ def anonymize(dicom_in, dicom_out,
                 folder_out = root.replace(wip_dicom_in, wip_dicom_out)
                 for name in files:
                     current_file = os.path.join(root, name)
-                    anonymize_file(current_file, folder_out,
-                                   tags_to_keep, tags_to_delete,
-                                   forced_values=forced_values,
-                                   anonymous=anonymous,
-                                   config_profile=config_profile)
+                    try:
+                        anonymize_file(current_file, folder_out,
+                                       tags_to_keep, tags_to_delete,
+                                       forced_values=forced_values,
+                                       anonymous=anonymous,
+                                       config_profile=config_profile)
+                    except AnonymizerError as e:
```

Comment:
Could it be a problem that all anonymisation errors are ignored ? For instance if it is a DICOM but something went wrong with the anonymization. If this is the intended behaviour, I would suggest to rename the `error_no_dicom` in something more general like `raise_an_error_if_any_file_cannot_be_properly_anonymized` or `anonymization_errors`.

Reply:
I don't think it's gonna be problem as `AnonymizerError` are only raised in case of DICOMDIR file or non DICOM file : 
https://github.com/cati-neuroimaging/deidentification/blob/118546224b9e408ac06dad7bff0181bf944deb73/deidentification/anonymizer.py#L495-L499
In other cases, there is some `DeidntificationError`, and for other errors no catched, it may raise the specific error.